cmd/user-agent/src/components/Tables/SimpleTable.vue

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
+
+  <div>
+    <md-table v-model="users" :table-header-color="tableHeaderColor">
+      <md-table-row slot="md-table-row"  slot-scope="{ item }"  style="text-align:left">
+        <badger-accordion>
+          <badger-accordion-item>
+            <template slot="header"><h5><md-icon>keyboard_arrow_down</md-icon>{{item.name}}</h5></template>
+            <template slot="content">{{item.credential}}</template>
+          </badger-accordion-item>
+        </badger-accordion>
+      </md-table-row>
+
+    </md-table>
+  </div>
+
+</template>
+<!--TODO : Fix the hard coded data with the actual data issue-89-->
+<script>
+  import {BadgerAccordion, BadgerAccordionItem} from 'vue-badger-accordion'
+
+  export default {
+    name: "simple-table",
+    components: {
+      BadgerAccordion,
+      BadgerAccordionItem
+    },
+    props: {
+      tableHeaderColor: {
+        type: String,
+        default: ""
+      }
+    },
+    data() {
+      return {
+        selected: [],
+        users: [
+          {
+            name: "Foo Student Card",
+            id: "1",
+            credential: "{\n" +
```

Comment:
This is the follow up task issue-89. I am already working on it. I wanted to break the task since it was getting huge